libs/core/tests/unit_tests/test_tools.py

```diff
@@ -1150,33 +1150,33 @@ class _MockStructuredToolWithRawOutput(BaseTool):
     name: str = "structured_api"
     args_schema: Type[BaseModel] = _MockSchema
     description: str = "A Structured Tool"
-    response_format: Literal["content_and_raw_output"] = "content_and_raw_output"
+    response_format: Literal["content_and_artifact"] = "content_and_artifact"
 
     def _run(
         self, arg1: int, arg2: bool, arg3: Optional[dict] = None
     ) -> Tuple[str, dict]:
         return f"{arg1} {arg2}", {"arg1": arg1, "arg2": arg2, "arg3": arg3}
 
 
-@tool("structured_api", response_format="content_and_raw_output")
-def _mock_structured_tool_with_raw_output(
+@tool("structured_api", response_format="content_and_artifact")
```

Comment:
alternative could be `with_artifact` a bit less to type, but less precise

Reply:
"content" vs "with_artifact" feels less clear to me than "content" vs "content_and_artifact"